src/utilis/Slider.tsx

```diff
@@ -0,0 +1,30 @@
+interface sliderProps {
```

Comment:
create sepererate file for every slider component and manage whole slider in one folder

Reply:
still not resolved